Tasks/NuGetInstaller/NuGetInstaller.ps1

```diff
@@ -2,9 +2,9 @@ param(
     [string]$solution,
     [ValidateSet("Restore", "Install")]
     [string]$restoreMode = "Restore",
-    [string]$excludeVersion,
+    [string]$excludeVersion, # Support for excludeVersion has been deprecated.
     [string]$noCache,
-    [string]$nuGetArgs,
```

Comment:
Seems like renaming this parameter is a bad idea. We aren't removing excludeVersion, but we have essentially removed this parameter. What's the difference? Should we deprecate the old name as well?